### DISCUSSION TITLE
How can I share variables via subscription?

### DISCUSSION THREAD

**Discussion by ghost** [hecrj/iced#855] [Q&A] [answered]
To make IRC GUI application, I use iced lib expecially download_progress example.
https://github.com/hecrj/iced/blob/master/examples/download_progress/src/download.rs

I've already made IRC GUI Reader using iced subscription, but I'd like to make sender client in subscription to send input message.
It's difficult to understand subscription functions fully.
Could you please give me any advices?

My work is here.
https://github.com/lilybrevec/gelato

**Comment by 13r0ck** [comment, marked as answer]
I don't understand your question. Communication across subscriptions is done with messages, and those messages can carry data.

**Comment by ghost** [comment]
I already avoided this problem by another way. Thanks!
